app/services/moves/import_people.rb

```diff
@@ -32,11 +32,13 @@ def import_people
 
         PersonalCareNeeds::Importer.new(profile: profile, personal_care_needs: personal_care_needs.fetch(offender_no, [])).call
 
-        if profile.changed?
-          changed_profile_count += 1
-          profile.save!
-        end
+        next unless profile.changed? || profile.person.changed?
+
+        changed_profile_count += 1
+        profile.person.save!
```

Comment:
We save the person first to make sure there is a person_id for the profile to be valid